source/chapters/appendix/mixxx_controls.rst

```diff
@@ -3174,6 +3174,26 @@ Note that :mixxx:coref:`[Library],MoveUp` and other Move and Scroll controls emu
    .. versionadded:: 2.1.0
 
 
+.. mixxx:control:: [Library],MoveTrackUp
+
+   Equivalent to pressing the :kbd:`Alt` + :kbd:`Up` key on the keyboard
+
+   :range: Binary
+   :feedback: Move selected track up
```

Comment:
or put it here (but maybe the explanation is to much for the 'feedback' hint):
> Track selection is moved up by one row

What do you think?

Reply:
I reworded it so it's a bit more like other entries